tests/src/test/java/org/eclipse/hono/tests/mqtt/MqttPublishTestBase.java

```diff
@@ -77,6 +85,7 @@
      *         message. The future will succeed if the message has been
      *         published successfully.
      */
+    // here
```

Comment:
???

Reply:
Thanks for spotting that was a "note" I overlooked before committing...